src/css/styles.css

```diff
@@ -1,5 +1,77 @@
-body, html {
+/* body, html {
   background-image: linear-gradient(to top, #ff9a9e 0%, #fecfef 99%, #fecfef 100%);
   background-repeat: no-repeat;
   height: 100%;
+} */
+
+body {
+  display: flex;
+  width: 2000px;
+  height: 1600px;
+  background-color: #050E2B;
+  color: #2A2B2C;
+}
+
+header {
```

Comment:
Good work trying to match as closely as you could to the wireframe! This is a good starting point to get some functionality going!

Reply:
Great work!